Tests/test_pairwise2_no_C.py

```diff
@@ -16,11 +16,18 @@
 
 from Bio import pairwise2
 
-# Explicitly using pure Python fallback functions:
-pairwise2._make_score_matrix_fast = pairwise2._python_make_score_matrix_fast
-pairwise2.rint = pairwise2._python_rint
-
```

Comment:
Did this have any functional usage? i.e. Might anyone be assuming these exist?

Reply:
No, this script does not get imported anywhere.

Reply:
Sorry, missed this was in the tests folder.